src/graphql/mutations/add-pending-withdraw.js

```diff
@@ -22,6 +22,12 @@ module.exports = async (root, data) => {
   const txReceipt = await getTransaction(txid);
   await DBHelper.insertTransactionReceipt(txReceipt);
 
+  const event = await DBHelper.findOneEvent({ address: eventAddress });
```

Comment:
please add comment above line

Reply:
please note that since we are adding to the list in the mutation (I assume you are doing this to show the pending status in the dashboard cards?), we need to handle if the withdraw tx fails when the tx gets confirmed. we should take it off the list in the event of a failed tx. see `sync/winnings-withdrawn.js L77`

Reply:
also this whole logic of adding it to the `withdrawnList` needs to be in the `sync/winnings-withdraw success` logic too. if we ever delete the db later and start from scratch, we want to populate the `withdrawnList` from all the withdraw events on chain.